docs/workflow.rst

```diff
@@ -36,30 +32,29 @@ https://code.teracy.org/projects/PILOT/repos/pixelate/browse project, follow the
 
 2. ``Clone`` it to   workspace.
 
-    **Step 1**: Click ``Clone`` to get the ``.git`` repository clone URL.
+    **Step 1**: Click ``Clone or download`` to get the ``.git`` repository clone URL.
 
-    **Step 2**: Copy the URL ``ssh://git@code.teracy.org/~hoavt/pixelate.git`` in the
+    **Step 2**: Copy the URL ``git@github.com:hoavt/teracy-dev.git`` in the
```

Comment:
Copy the URL in the SSH field (for example, mine is git@github.com:hoavt/teracy-dev.git)